crates/router/src/connector/worldpay.rs

```diff
@@ -698,7 +698,7 @@ impl api::IncomingWebhook for Worldpay {
             EventType::Error | EventType::Expired => {
                 Ok(api::IncomingWebhookEvent::PaymentIntentFailure)
             }
-            _ => Err(errors::ConnectorError::WebhookEventTypeNotFound.into()),
+            _ => Ok(api::IncomingWebhookEvent::EventNotSupported),
```

Comment:
Avoid matching on `_`.

Reply:
In this connector  also if we remove the match on `_` , there would be an error for the missing match arms for the other fields in `EventType` enum

Reply:
Please match on all variants explicitly, you can have it validated by @jagan-jaya or @ArjunKarthik if needed.

The reason I'm insistent on matching all variants explicitly is that a similar wildcard match was the cause of an incorrect error mapping that was observed on production.